src/svgp.jl

```diff
@@ -0,0 +1,68 @@
+struct SVGP end
+
+raw"""
+    approx_posterior(::SVGP, fz::FiniteGP, q::MvNormal)
```

Comment:
Maybe this is better turned into its own issue: Have we hardcoded that "inducing variable" is only u = f(Z)? How would we support generic inducing variables (Fourier features, spherical harmonics, or even conditioning on gradients)?

Reply:
Opened #11